drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -116,6 +116,7 @@
                 <h3>{% trans "Items" %}</h3>
             </header>
             <section class="tab-content">
+                <div class="xblock-editor-error-messages" aria-live="polite" tabindex="0"></div>
```

Comment:
@cahrens No, it's not generic - those method collects data from all the "tabs" and sends it; each individual tab does not send anything. The validation on previous tabs are client-side only (btw, I don't like it as well - it is relatively easy to workaround that).

Putting the error next to control is a sensible suggestion, and actually I evaluated that option first; unfortunately, it would require somewhat involving overhaul of editor, so, taking into account deadlines, my vacation and other things I've decided that current solution would be the best I could reliably deliver. 

To explain this decision a bit further, in this particular case those errors are caused by groups of checkboxes that are part of "item" block; there are always more than one such "item" blocks; and so far they are indistinguishable from each other. So, to implement that suggestion I would have to assign some identifiers to the fields, or at least "item" blocks + make sure backend validation returns errors associated with those identifiers somehow.

Since this task is not about improving author experience, I decided to stop right there and follow a simpler approach.

Also, I really love how me being proactive about some possible problems brings just more and more feedback, requesting my little improvement not even mentioned in the task description/acceptance criteria (no one thought about that) to be absolutely bulletproof :)

Also, I've just finished a11y course training, and one of the recommendations was to put errors summary *in the beginning* of the form, so I'll put it there - in the beginning of the form. Visually it wouldn't make any difference though.

Should we bring someone from UX team maybe?